src/imex.rs

```diff
@@ -291,63 +293,90 @@ pub(crate) async fn import_backup_stream<R: tokio::io::AsyncRead + Unpin>(
     backup_file: R,
     file_size: u64,
     passphrase: String,
-) -> Result<()> {
+) -> (Result<()>,) {
```

Comment:
This is to avoid careless use of the `?` operator to not skip the cleanup

Reply:
why not use a drop guard to ensure cleanup?

Reply:
Then sync `remove_file()` must be used instead of `tokio::fs::remove_file()` which looks weird for me in async code. Also blobs must be removed only on error, but the unpacked db in any case, so there will be two guards one of which should be canceled. I think that won't simplify the code. The only pro of using drop guards is that the cleanup is done even if the future is canceled, but if the program is killed, the cleanup isn't done anyway

Reply:
Reverted the interface change, added `import_backup_stream_inner()` instead